docs/source/distributed_systems.rst

```diff
@@ -26,19 +26,18 @@ Binding with distributed systems
 ================================
 
 Working with distributed systems is easy, you just basically need to make
-sure you use TCP transport and to specify the network interface where you
-want to bind.
+sure you use TCP transport and to specify the IP address that you want to bind
+to.
 
 This can be done when binding (note that we are using `127.0.0.1` but you
-may use any other interface available in your system):
+may use any other address available in your system):
```

Comment:
Agree with this change: address is better than interface. However, maybe it is good to point out that the IP address cannot be any IP address, but only those corresponding to your network interfaces.

What do you think about using "network interface IP address" instead. Just for completeness. :blush:

Reply:
Sounds good to me.